src/rgw/CMakeLists.txt

```diff
@@ -210,6 +213,9 @@ target_include_directories(rgw_a SYSTEM PUBLIC "../rapidjson/include")
 if(WITH_RADOSGW_AMQP_ENDPOINT)
   find_package(RabbitMQ REQUIRED)
 endif()
+if(WITH_RADOSGW_KAFKA_ENDPOINT)
+  find_package(RDKafka REQUIRED)
```

Comment:
you can specify the minimum version here so that attempts to build against older versions will fail during configuration (with an obvious error message) instead of during build/link